k8s/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "k8s",
+  "version": "1.0.0",
+  "main": "main.js",
+  "types": "main.ts",
+  "license": "Apache-2.0",
+  "private": true,
+  "scripts": {
+    "import": "cdk8s import",
+    "synth": "cdk8s synth",
+    "compile": "tsc",
+    "watch": "tsc -w",
+    "test": "jest",
```

Comment:
also minor but not sure how this command is being used, not seeing any tests or jest config

Reply:
Good catch, jest was added automatically when creating the project but deleted later. Thanks!